README.md

```diff
@@ -1,6 +1,6 @@
 [![Slack Status](http://sdlslack.herokuapp.com/badge.svg)](http://slack.smartdevicelink.org)
 
-[![Build Status](https://travis-ci.org/smartdevicelink/sdl_core.svg?branch=master)](https://travis-ci.org/smartdevicelink/sdl_core)  **LINUX**
+[![Build Status](https://travis-ci.org/smartdevicelink/sdl_core.svg?branch=develop)](https://travis-ci.org/smartdevicelink/sdl_core)  **LINUX**
```

Comment:
@LuxoftAKutsan Please make the following changes:
1)  **LINUX** `->` **Ubuntu Linux 14.04 x64**
2) **Windows native** `->` **Windows 7 x64** 
3) **QT** `->` **Windows 7 x86 QT**